### DISCUSSION TITLE
Do I need to periodically force a full backup like I would with Deja-Dup or duplicity?

### DISCUSSION THREAD

**Discussion by cxr6548** [borgbackup/borg#6330] [Q&A] [answered]
Duplicity recommends periodically making a full backup in case something goes wrong in the incremental chain so that data loss is minimized. Do I need to do something like this in borg or is backing up handled in a fundamentally different way that makes this unnecessary?

**Comment by infectormp** [comment, marked as answer]
https://borgbackup.readthedocs.io/en/stable/faq.html#if-a-backup-stops-mid-way-does-the-already-backed-up-data-stay-there

> Borg always does full single-pass backups, so it will start again from the beginning - but it will be much faster, because some of the data was already stored into the repo (and is still referenced by the checkpoint archive), so it does not need to get transmitted and stored again.

**Comment by ThomasWaldmann** [comment]
Yes, it's fundamentally different. Logically, each borg backup archive is a FULL backup.

So, there are no chains. Also, you can delete any backup archive you want without considering its "relationship" to other backup archives (each archive is independent of other archives, that also makes `borg prune` much simpler).

**Comment by andrewcrook** [comment]


**Reply by RonnyPfannschmidt** [reply]
Why push a contextually wrong answer into a discussion thats practically solved since about a year

**Reply by RonnyPfannschmidt** [reply]
The explanation of the function implies the answer op was searching for while you answer a completely different question

**Reply by andrewcrook** [reply]
I  don’t agree but I don't have the energy to argue with something so trivial. I see someone else has since marked the answer rather than the actual op anyway.  So I have removed the comments to draw a line under it and move on.
